pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -48,6 +48,7 @@ type MigPlanSpec struct {
 	DestMigClusterRef *kapi.ObjectReference `json:"destMigClusterRef,omitempty"`
 	MigStorageRef     *kapi.ObjectReference `json:"migStorageRef,omitempty"`
 	Closed            bool                  `json:"closed,omitempty"`
+	UnhealthyResources
```

Comment:
Nit: For consistency, let's group _inline_ structs at the beginning of the struct definition.

Reply:
done